app/views/comment_notifier/notify_councillor.html.haml

```diff
@@ -1,27 +1,30 @@
-%p Hi #{@comment.councillor.name},
+= comment_as_html(@comment.text)
+
+%p From #{@comment.name}
+
+%hr
+
+%p #{@comment.name} posted this message to you on PlanningAlerts in response to a local planning application.
+
+%p You can publicly respond by simply replying to this email.
 
 %p
-  You’ve got a new message from
-  = @comment.name
-  in relation to a
-  = succeed "." do
-    = link_to "local planning application for #{@comment.application.address}",
+  The planning application is for
+  = succeed ":" do
+    = link_to "#{@comment.application.address}",
               application_url(@comment.application,
                               protocol: "https",
                               utm_source: "councillor-notifications",
                               utm_medium: "email",
                               utm_campaign: "view-application")
-  The application is for, “#{@comment.application.description}”.
+  “#{@comment.application.description}”.
 
-%p
-  = @comment.name
-  writes:
-
-%blockquote= comment_as_html(@comment.text)
-
-%p
-  This message was posted publicly on PlanningAlerts.
-  You can publicly respond to this message by simply replying to this email.
+%p= link_to "Learn more about the application and see what others have to say at PlanningAlerts.",
+            application_url(@comment.application,
+                            protocol: "https",
```

Comment:
Yeah I think fixing that is for another day. It should be using the url helpers that we use in alerts, but it needs to know about a few extra things and those helpers might need to be make more generic.